src/Bicep.Cli/Program.cs

```diff
@@ -26,17 +26,20 @@ public class Program
         private readonly TextWriter errorWriter;
         private readonly IResourceTypeProvider resourceTypeProvider;
 
-        public Program(IResourceTypeProvider resourceTypeProvider, TextWriter outputWriter, TextWriter errorWriter)
+        private readonly string assemblyFileVersion;
+
+        public Program(IResourceTypeProvider resourceTypeProvider, TextWriter outputWriter, TextWriter errorWriter, string assemblyFileVersion)
         {
             this.resourceTypeProvider = resourceTypeProvider;
             this.outputWriter = outputWriter;
             this.errorWriter = errorWriter;
+            this.assemblyFileVersion = assemblyFileVersion;
         }
 
         public static int Main(string[] args)
         {
             BicepDeploymentsInterop.Initialize();
-            var program = new Program(new AzResourceTypeProvider(), Console.Out, Console.Error);
+            var program = new Program(new AzResourceTypeProvider(), Console.Out, Console.Error, ThisAssembly.AssemblyFileVersion);
```

Comment:
use `this.assemblyFileVersion`

Reply:
This is the `Main` method, which is only run in "production". `this` doesn't exist here

Reply:
Oops, yeah I missed that :)